<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="txt_title">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RSA Cipher</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_subtitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Get your keys or enter your own below</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_getkey">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Get keys</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_public_key">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PUBLIC KEY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_get_publickeyvalue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_get_privatekeyvalue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_enter_msg">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your message :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_msg">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>741</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_encrypt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>700</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output_msg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>786</y>
     <width>751</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_output_msg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>740</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your message encrypted:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_decrypt">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>700</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_private_key">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PRIVATE KEY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="d_get_privatekeyvalue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_get_publickeyvalue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_publickey_n">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>n :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_privatekey_n">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>n :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_privatekey_d">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>d :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_publickey_e">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>e :</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_enter_keys">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your keys :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="n_inputvalue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_n_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>n :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_inputvalue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_d_input">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>d :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="d_inputvalue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txt_e_input">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>e :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
